packages/flutter/lib/src/services/text_layout_metrics.dart

```diff
@@ -54,6 +54,24 @@ abstract class TextLayoutMetrics {
     return true;
   }
 
+  /// Check if the given code unit is a line terminator character.
+  ///
+  /// Includes newline characters from ASCII
+  /// (https://www.unicode.org/standard/reports/tr13/tr13-5.html).
+  static bool isLineTerminator(int codeUnit) {
+    switch (codeUnit) {
+      case 0xA: // line feed
+      case 0xB: // vertical feed
+      case 0xC: // form feed
+      case 0xD: // carriage return
```

Comment:
When there is a CRLF you shouldn't break after CR

Reply:
Do you mean the CR should come before the LF in this switch?

Reply:
huh the unicode website is offline. You should not break after CR if it's followed by a LF.

Reply:
Aren't 0x2028 and 0x2029 also supposed to be on this list?

Reply:
Done, thanks for catching that!